akka-actor-tests/src/test/scala/akka/routing/MetricsBasedResizerSpec.scala

```diff
@@ -244,8 +244,8 @@ class MetricsBasedResizerSpec extends AkkaSpec(ResizerSpec.config) with DefaultT
 
       resizer.reportMessageCount(router.routees, router.msgs.size)
 
-      val after = LocalDateTime.now
-      resizer.performanceLog(2).toMillis shouldBe (java.time.Duration.between(before, after).toMillis / 2 +- 1)
+      val after = System.nanoTime()
+      resizer.performanceLog(2).toMillis shouldBe ((before - after).nanos.toMillis / 2 +- 1)
```

Comment:
Yeah, I thought it was safe because of actually measuring, and basing on that, but, even so it would need an explicit sleep in-between the two operations to make sure it is not done super-quickly.

So maybe that plus the 'give-or-take' part as a factor of the total time rather, so that it is more forgiving on a slow machine...